src/test/scala/cromwell/CromwellTestkitSpec.scala

```diff
@@ -134,14 +134,23 @@ with DefaultTimeout with ImplicitSender with WordSpecLike with Matchers with Bef
     TestActorRef(new WorkflowManagerActor(dataAccess, new LocalBackend))
   }
 
-  def runWdl(fsm: TestFSMRef[WorkflowState, WorkflowActor.WorkflowFailure, WorkflowActor], eventFilter: EventFilter, expectedOutputs: Map[FullyQualifiedName, WdlValue] = Map.empty): Unit = {
+  def runWdl(fsm: TestFSMRef[WorkflowState, WorkflowActor.WorkflowFailure, WorkflowActor],
+             eventFilter: EventFilter,
+             expectedOutputs: Map[FullyQualifiedName, WdlValue] = Map.empty,
+             terminalState: WorkflowState = WorkflowSucceeded): Unit = {
     assert(fsm.stateName == WorkflowSubmitted)
     eventFilter.intercept {
       fsm ! WorkflowActor.Start
       within(timeoutDuration) {
         awaitCond(fsm.stateName == WorkflowRunning)
-        awaitCond(fsm.stateName.isTerminal)
-        fsm.stateData should be(WorkflowActor.NoFailureMessage)
+        awaitCond(fsm.stateName == terminalState)
+
+        if (terminalState == WorkflowSucceeded)
+          fsm.stateData should be(WorkflowActor.NoFailureMessage)
+
+        if (terminalState == WorkflowFailed)
+          fsm.stateData.isInstanceOf[WorkflowActor.FailureMessage] shouldEqual true
```

Comment:
yeaaaaaaah.... I was hoping nobody noticed that :-D

The reason was that if I tried to do `fsm.stateData should be(WorkflowActor.FailureMessage)`, it wanted me to construct FailureMessage with a String and I just wanted to assert that it failed and I didn't care about the error.

Maybe there's a better way for me express that though.